javascript/isOdd.js

```diff
@@ -0,0 +1 @@
+const isOdd = n => n & 1
```

Comment:
```suggestion
const isOdd = n => !!(n & 1)
```
your usage example shows strict equality comparison between the return value of the function and a boolean, so maybe prepending !! to convert 0/1 to a boolean would fix